cmd/kots/cli/install.go

```diff
@@ -224,39 +225,61 @@ func InstallCmd() *cobra.Command {
 			}
 
 			log.ActionWithoutSpinner("Deploying Admin Console")
+			// Initialize kots metrics before Deploy to monitor its success/failure
+			// Enable sampling only in non-airgap mode
+			var cause string
+			im, err := InitMetrics(&deployOptions, isAirgap)
+			if err != nil {
+				return errors.Wrap(err, "failed to init install metrics")
```

Comment:
This should not error out.  Installation should proceed if metrics cannot be reported.  However, if this fails, sending updates becomes unnecessary.

Reply:
will do log.Info and proceed.